.env.example

```
@@ -1,2 +1,4 @@
 PORT=8080
-LOG_LEVEL= debug
\ No newline at end of file
+LOG_LEVEL= debug
+DATABASE_URL="mongodb+srv://anumahjoshuaeneye:zkLXwn3J3VVMcLVg@cluster0.bi0jnzn.mongodb.net/shortenIt?retryWrites=true&w=majority&appName=Cluster0"
+BASE_URL = 'https://shortit';
```

You should not push your database credentials to git

```suggestion
LOG_LEVEL=debug
DATABASE_URL=
BASE_URL='https://shortit';
```

---

Done